service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -74,7 +70,9 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
 
         if (errors.isEmpty()) {
             try {
-                BigDecimal amount = getAmountToPay(caseData.getOrders());
+                List<OrderType> orderTypes = caseData.getOrders() == null ? null : caseData.getOrders().getOrderType();
```

Comment:
We could do `Optional.ofNullable(caseData.getOrders()).map(Orders::getOrderType)` here and pass it to `getFeeAmountForOrders` method, where `ofNullable` could be removed.

Reply:
nevermind, did my homework, it's not a good practice to use Optionals as method params 😄 resolving